common/src/types/tasks.rs

```diff
@@ -422,6 +428,31 @@ impl From<PayRelayerFeeTaskDescriptor> for TaskDescriptor {
     }
 }
 
+/// The task descriptor for redeeming a relayer note
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct RedeemRelayerFeeTaskDescriptor {
+    /// The wallet ID of the relayer's wallet
+    ///
+    /// Technically this should be static and not needed here, but we include it
+    /// to allow the descriptor struct to compute its own task queue key
+    pub wallet_id: WalletIdentifier,
```

Comment:
why can't we use a `static` in computing the task queue key?

Reply:
this value isn't static, it may be generated on startup

Reply:
ah, blanked out - we'd have to fetch this from state, right, which is async and doesn't play well with the `get_task_queue_key` interface?

Reply:
i suppose we could put it in a `OnceCell` on startup, but eh, inconsequential